content/collections/experiment/en/overview.md

```diff
@@ -8,103 +8,110 @@ this_article_will_help_you:
+The most common bucketing unit is “user”. However, if you’re a B2B business or are utilizing the collaboration feature, you might want to use a bucketing unit such as “organization” or company\_id, which means that every user within the same organization could see the same variant. This can help reduce product-related confusion caused by disparate user interfaces if people are sitting next to their coworkers. Another reason for bucketing by company\_id is to reduce the load on your customer support team. It's easier for the customer support team to know which accounts have which features enabled. Either way, you want to make sure the Stable Unit Treatment Value Assumption ([SUTVA](https://blogs.iq.harvard.edu/violations_of_s#:~:text=Methods%20for%20causal%20inference%2C%20in,treatments%20of%20others%20around%20him)) holds for whatever bucketing unit you choose to best ensure inference. 
 
 {{partial:admonition type='note'}}
  If your organization has purchased the [Accounts add-on,](/docs/analytics/account-level-reporting) you may perform bucketing and analysis on groups rather than users.
 {{/partial:admonition}}
 
-### Allocate users
+## Allocate users
 
-Now that you’ve got your variants, you’ll need to decide how many of your users will see them. You can choose to roll your experiment out to your entire user base, or you can just roll it out to a fraction of them instead. You’ll specify how many users in your experiment will see the control experience and how many will see your variants. You can define specific user segments to include or exclude from your experiment, and you can even choose a specific experience for individual user or device IDs.
+Decide how many users should see your variants by rolling the experiment out to the entire user base or just a fraction. Specify the number of users for the control and variant experiences, define user segments to include or exclude, and even target specific user or device IDs.
 
-### Activate your experiment
+## Activate your experiment and analyze your results
 
-At this point, you’re ready to roll out your experiment to your users. Click *Start Experiment*, and your experiment will be live.
+When your experiment is set up, you can select *Start Experiments* to send it to live users.
```

Comment:
```suggestion
When your experiment is set up,  click *Start Experiments* to send it to live users.
```

nit: "you can" is unnecessary places like this.

Also, this doesn't really send experiments to users, but rather enables the experiment so that users can be exposed to it.